Description: Fix CMake project version for 0.4.12 release
Forwarded: https://github.com/qualcomm/userspace-resource-manager/pull/338
Author: Kartik Nema <kartnema@qti.qualcomm.com>
Last-Update: 2026-10-01

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 62bbb8f..34f3ecb 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -5,7 +5,7 @@ set(CMAKE_STANDARD_REQUIRED True)
 set(CMAKE_POSITION_INDEPENDENT_CODE ON)
 add_compile_options(-Wall -Wextra)
 
-project("URM" VERSION 0.4.9)
+project("URM" VERSION 0.4.12)
 
 include(GNUInstallDirs)
 find_package(PkgConfig REQUIRED)
